.golangci.yml

```diff
@@ -94,6 +94,14 @@ linters-settings:
       - 'math.*'
       - 'strconv.ParseInt'
 
+  dupword:
```

Comment:
q: I have noticed that this linter caught more than one error from when we format the comments to do not have more than 80 columns per line. Any reason to constraint it to the three common words that we use here?

Reply:
Glad you asked! This is simply from the comments below,
```
lntest/itest/lnd_switch_test.go:24:1: Duplicate words (X) found (dupword)
//  2. -------X       X         X   Bob  restart sender and intermediaries
```

And other similar places. My take is that we can skip linting those places(using `nolint:dupword`), or, we can only check for common words only. Took the lazy approach here, but I was NOT sure about it.